### ISSUE TITLE
dvs_msgs/EventDepthArray.h

### ISSUE THREAD

**Issue by hebaoy** [huangxueyan/PEME#2]
don't find file <dvs_msgs/EventDepthArray.h> in "rpg_dvs_ros" package

**Comment by aelmoudni** [comment]
Yes, I have the same problem, i have already been working with the dvs_msgs drivers and i know that files such as Event.h and EventArray.h but the EventDepthArray.h is non existing

**Comment by PauloHLemos** [comment]
Did anyone find a fix.

**Comment by huangxueyan** [comment]
I haven't saved this file now. EventDepthArray is quite simple, and you can implement it yourself. 
Additionally, I have made my experimental code public. The code might be a bit messy, but I hope it can be helpful to you. https://github.com/huangxueyan/hxy-rotation
